python/lsst/summit/utils/utils.py

```diff
@@ -308,6 +310,80 @@ def detectObjectsInExp(exp, nSigma=10, nPixMin=10, grow=0):
     return footPrintSet
 
 
+def fluxesFromFootprints(footprints, parentImage, subtractImageMedian=False):
+    """Calculate the flux from a set of footprints, given the parent image,
+    optionally subtracting the whole-image median from each pixel as a very
+    rough background subtraction.
+
+    Parameters
+    ----------
+    footprints : `lsst.afw.detection.FootprintSet` or
+                 `lsst.afw.detection.Footprint` or
+                 `iterable` of `lsst.afw.detection.Footprint`
+        The footprints to measure.
+    parentImage : `lsst.afw.image.Image`
+        The parent image.
+    subtractImageMedian : `bool`, optional
+        Subtract a whole-image median from each pixel in the footprint when
+        summing as a very crude background subtraction. Does not change the
+        original image.
+
+    Returns
+    -------
+    fluxes : `list` of `float`
+        The fluxes for each footprint.
```

Comment:
Is there any value in returning a numpy array here instead?

Reply:
I'm happy with a list, personally. If the user needs, it can always be cast with `np.asarray` as needed.

Reply:
I can't see the harm. I'll implement unless @mfisherlevine objects.

Reply:
I can't see the harm. I'll implement unless @mfisherlevine objects.